syntax = "proto3";

package api.user.request;

message DeleteConnection {
    uint64 user_id = 1;
}
